testing/cloudi/APKBUILD

```diff
@@ -0,0 +1,91 @@
+# Contributor: Michael Truog <mjtruog@gmail.com>
+# Maintainer: Michael Truog <mjtruog@gmail.com>
+pkgname=cloudi
+pkgver=1.7.1.1
+_srcver=2b9a0bfe44433fdcb1bebb684fe436c9b83af4b1
+pkgrel=0
+pkgdesc="Cloud computing framework for efficient, scalable, and stable soft-realtime event processing."
+url="http://cloudi.org/"
+license="MIT"
+arch="all"
+depends="g++"
+makedepends="autoconf
+             automake
+             binutils-dev
+             boost-dev
+             boost-system
+             boost-thread
+             erlang
+             erlang-asn1
+             erlang-common-test
+             erlang-crypto
+             erlang-dev
+             erlang-erl-interface
+             erlang-eunit
+             erlang-inets
+             erlang-jinterface
+             erlang-public-key
+             erlang-reltool
+             erlang-sasl
+             erlang-snmp
+             erlang-ssl
+             erlang-syntax-tools
+             erlang-tools
+             erlang-xmerl
+             gmp-dev
+             libexecinfo-dev
+             libtool
+             nodejs
+             openjdk8
+             perl
+             php7
+             python2
+             python2-dev
+             ruby
+             "
+install=""
+subpackages=""
+source="https://github.com/$pkgname/$pkgname/archive/$_srcver.tar.gz
```

Comment:
I switched this from the pkgver to a git rev for testing purposes only.  That allows the use of recent changes for the pidfile path provided to configure and newer cloudi script use.  I went over the older (`1.7.1` release and previous) `cloudi` script and the `nodetool` script to simplify and improve the tasks they do.  The one thing that remains is that distributed Erlang is used by CloudI and must be started.  CloudI starts epmd if it hasn't been started already, but that isn't ideal as a long-term solution, that is only a way of making sure stuff always works in any deployment scenario.  The `community/erlang` package really needs a script to handle epmd startup, so it is given more visibility as a dependency, though that is outside the scope of this pull request (this package works either way).